### PR TITLE
[quic] add `send_response_header` event

### PR DESCRIPTION
**Pull request by gfx**
To see H2O response headers.

example output:

```
{"at":1582258275564,"type":"send_response_header","master_conn_id":0,"h2o_conn_id":11,"h2o_req_id":0,"h2o_header_name":"content-length","h2o_header_value":"177"}
{"at":1582258275564,"type":"send_response_header","master_conn_id":0,"h2o_conn_id":11,"h2o_req_id":0,"h2o_header_name":"date","h2o_header_value":"Fri, 21 Feb 2020 04:11:15 GMT"}
{"at":1582258275565,"type":"send_response_header","master_conn_id":0,"h2o_conn_id":11,"h2o_req_id":0,"h2o_header_name":"content-type","h2o_header_value":"text/html"}
{"at":1582258275565,"type":"send_response_header","master_conn_id":0,"h2o_conn_id":11,"h2o_req_id":0,"h2o_header_name":"last-modified","h2o_header_value":"Mon, 10 Feb 2020 03:41:35 GMT"}
{"at":1582258275565,"type":"send_response_header","master_conn_id":0,"h2o_conn_id":11,"h2o_req_id":0,"h2o_header_name":"etag","h2o_header_value":"\"5e40d0ef-b1\""}
{"at":1582258275565,"type":"send_response_header","master_conn_id":0,"h2o_conn_id":11,"h2o_req_id":0,"h2o_header_name":"accept-ranges","h2o_header_value":"bytes"}
{"at":1582258275565,"type":"send_response_header","master_conn_id":0,"h2o_conn_id":11,"h2o_req_id":0,"h2o_header_name":"alt-svc","h2o_header_value":"h3-25=\":8081\""}
```

### CODE REVIEW

File: h2olog [hunk 1]
```diff
@@ -644,9 +678,13 @@ def handle_resp_line(cpu, data, size):
 def load_common_fields(hsh, line):
     for k in ['at', 'type', 'master_conn_id']:
         v = getattr(line, k)
-        # FIXME: This hack is no longer needed as of: https://github.com/h2o/quicly/pull/279
+        # only quicly's probes have `at`
```
Comment (gfx) [inline h2olog]: This is no longer a dirty-hack because H2O probes have no `at` in design.

File: h2olog [hunk 2]
```diff
@@ -627,6 +636,31 @@ int trace_quictrace_lost(struct pt_regs *ctx) {
 
     return 0;
 }
+
+// send_response_header(uint64_t conn_id, uint64_t req_id, const char *name, size_t name_len, const char *value, size_t value_len)
```
Comment (toru) [inline h2olog]: Is this a memo that you were using, that you forgot to delete? (I do the same thing)

**Comment by toru** [comment]
Thank you! this looks like a good starting point. In the next iteration, I'd like to make this an optional behavior because this trace is technically not at the QUIC layer. So, something like `-v` for verbose, or `-r` for req/resp.

File: h2olog (hunk 2 above)
Comment (gfx) [reply h2olog]: Yes, it was just a memo. Removed 58a672b2714c9fe7a32a60126f916ad7cfed3db4

**Comment by gfx** [comment]
`-v` seems good! 

However, `-r` is ambiguous because req and resp are completely different data. How about `-q` for reQuest filters and `-p` for resPonse filters? We need only `-p` for now, though.

**Pull request merged**